### ISSUE TITLE
Writing to custom schema?

### ISSUE THREAD

**Issue by quinnhornblow** [fal-ai/fal#441]
Hi, I'm having trouble getting fal to place my models in the right place. I have a dbt_project.yml like so:

```
models:
  project:
    records_tables:
      +schema: records
```

all my classic dbt models in the models/record_tables/ directory get placed into <target_schema>_<custom_schema> as discussed [here](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas).

With `fal flow run --experimental-models`, fal models in the same directory get placed into <target_schema>, ignoring the custom schema.

I am using the write_to_model function, should I be using the write_to_source function?

**Comment by chamini2** [comment]
Hello, @quinnhornblow. I think we had not considered this use case before. In this case, changing to `write_to_soure` would not fix this. Let us take a look into supporting this. Thanks for the report!

**Comment by chamini2** [comment]
Can you paste the output of the commands

```
fal --version
dbt --version
```

I have just tested this in our main branch and it seems to be working as expected for me with the postgres adapter.

```
                 List of relations
     Schema     |      Name       | Type  | Owner
----------------+-----------------+-------+--------
 dbt_fal_custom | fal_008_model_a | view  | pguser
 dbt_fal_custom | fal_008_model_b | view  | pguser
 dbt_fal_custom | fal_008_model_c | table | pguser
 dbt_fal_custom | fal_008_model_d | view  | pguser
 dbt_fal_custom | fal_008_model_e | table | pguser
```

Both `fal_008_model_e` and `fal_008_model_c` are fal models.

**Comment by chamini2** [comment]
What I am seeing is that we have a bug reading data from a custom schema (https://github.com/fal-ai/fal/pull/442) I will fix that in the meantime.

**Comment by quinnhornblow** [comment]
> Can you paste the output of the commands
> 
> ```
> fal --version
> dbt --version
> ```

fal = 0.3.4
dbt = 1.1.0

Thanks for looking into this

**Comment by quinnhornblow** [comment]
I discovered that fal was writing to a redundant schema which I now deleted. Now It raises an error saying schema "" does not exist. That fix you mentioned above might do it.
Does fal use the process for generating a model name as [this](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas#how-does-dbt-generate-a-models-schema-name) macro?

**Comment by chamini2** [comment]
> Does fal use the process for generating a model name as [this](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas#how-does-dbt-generate-a-models-schema-name) macro?

It doesn't we get it from dbt and expect it to have it's correct value already.

**Comment by chamini2** [comment]
I think #442 will fix what you are seeing. I will let you know as soon as we release.

**Comment by chamini2** [comment]
Hello, @quinnhornblow . We have released [version 0.3.6](https://github.com/fal-ai/fal/releases/tag/v0.3.6) and it includes #442. Let me know if you have any problems, but I would expect this to be solved. You can reopen this if the behavior is not fixed or you find any other problem!
